index.html

```diff
@@ -15,12 +15,14 @@
     <script src="node_modules/chart.js/dist/Chart.js"></script>
     <script src="node_modules/urlhash/urlHash.js"></script>
     <script src="dist/community-toolbox.js" charset="utf-8"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.4.2/js/swiper.min.js"></script>
```

Comment:
Can this be included through `node_modules` or added to the repo instead of a CDN?

Reply:
it does not affect much until user is working offline
besides that both way are same

Reply:
What if a developer is trying to contribute but inet is not working? The lib will also not work and might even break the page by showing an error.